lib/utils/docker.go

```diff
@@ -38,6 +38,11 @@ func dockerLogin(username string, password string) {
 	log.Println("Logged into Harbor successfully")
 }
 
+func CheckDockerfile(_DockerfilePath string) bool {
```

Comment:
Not needed, stat the files wherever it is required directly

Reply:
done

Reply:
Still present